src/Sparrow/Platform/PlatformDetails.cs

```diff
@@ -19,6 +19,7 @@ public static class PlatformDetails
         public static readonly bool RunningOnLinux = RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
 
         public static readonly bool CanPrefetch = IsWindows8OrNewer() || RunningOnPosix;
+        public static readonly bool CanDiscardMemory = IsWindows8OrNewer() || RunningOnPosix;
```

Comment:
DiscardVirtualMemory can be used in Windows 8.1 and Server 2012 R2 or newer.

You need version 6.3 or higher.